iquip/apps/explorer.py

```diff
@@ -113,19 +156,20 @@ def openExperiment(self):
         experimentFileItem = self.explorerFrame.fileTree.currentItem()
         if experimentFileItem.childCount():
             return
-        experimentPath = self.getFullPath(experimentFileItem)  # pylint: disable=unused-variable
+        experimentPath = self.fullPath(experimentFileItem)  # pylint: disable=unused-variable
 
-    def getFullPath(self, experimentFileItem: QTreeWidgetItem) -> str:
+    def fullPath(self, experimentFileItem: QTreeWidgetItem) -> str:
         """Finds the full path of the file item and returns it.
 
         Args:
             experimentFileItem: The file item to get its full path.
         """
-        path = experimentFileItem.text(0)
+        path = [experimentFileItem.text(0)]
         while experimentFileItem.parent():
             experimentFileItem = experimentFileItem.parent()
-            path = posixpath.join(experimentFileItem.text(0), path)
-        path = posixpath.join(self.repositoryPath, path)
+            path.append(experimentFileItem.text(0))
+        path.append(self.repositoryPath)
+        path = posixpath.join(*reversed(path))
         return path
```

Comment:
Actually I prefer not to bind a new value of different types to a variable 😅
Could you just return it directly or change the variable name?
For example:
```python
return posixpath.join(*reversed(path))
```
or
```python
paths = [...]
...
path = ...
return path
```

Reply:
Ah😃 Then, I will return it directly.
Also, I think it would be better to change the former `path` to `paths` since it contains lots of the paths (even if parts).